keanu-project/src/main/java/io/improbable/keanu/algorithms/NetworkSamples.java

```diff
@@ -106,6 +110,18 @@ public double probability(Function<NetworkState, Boolean> predicate) {
         return (double) trueCount / networkStates.size();
     }
 
+    public List<Object> getSample(int sample) {
```

Comment:
Should this return a NetworkState at a specified sample number? Otherwise how can the user know with object belongs to which RV/operation result?